packages/lesswrong/components/posts/PostsPage/PostsPodcastPlayer.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import NoSSR from '@material-ui/core/NoSsr';
+import { registerComponent } from '../../../lib/vulcan-lib';
+import { applePodcastIcon } from '../../icons/ApplePodcastIcon';
+import { spotifyPodcastIcon } from '../../icons/SpotifyPodcastIcon';
+import { isClient } from '../../../lib/executionEnvironment';
+
+const styles = (): JssStyles => ({
+  embeddedPlayer: {
+    marginBottom: '2px'
+  },
+  podcastIconList: {
+    paddingLeft: '0px',
+    marginTop: '0px'
+  },
+  podcastIcon: {
+    display: 'inline-block',
+    marginRight: '8px'
+  }
+});
+
+const PostsPodcastPlayer = ({ podcastEpisode, classes }: {
+  podcastEpisode: PostsBase_podcastEpisode,
+  classes: ClassesType
+}) => {
+  const embedScriptFunction = (src: string, clientDocument: Document) => <>{
+    function(d) {
+      const x = d.getElementById('buzzsproutPlayerScript');
+      if (x) x.parentNode?.removeChild(x);
+      const s=d.createElement('script');
+      d.head.appendChild((
+        s.async=true,
+        s.src=src,
+        s.id='buzzsproutPlayerScript',
+        s
+      ));
+    }(clientDocument)
```

Comment:
Is this copy-pasta'd from something? I'm not a fan of the style, for reasons I expect you to share. Maybe link the source if it is? If you feel comfortable, consider writing it in a more FM-idiomatic style.

Reply:
Mostly, yeah.  I think it was originally minified; that probably doesn't matter given our build process.  Will re-write